firefox-ios/Client/Frontend/Settings/Main/Account/SyncNowSetting.swift

```diff
@@ -49,11 +49,7 @@ class SyncNowSetting: WithAccountSetting {
                 string: .FxANoInternetConnection,
                 attributes: [
                     NSAttributedString.Key.foregroundColor: theme.colors.textWarning,
-                    NSAttributedString.Key.font: DefaultDynamicFontHelper.preferredFont(
-                        withTextStyle: .body,
-                        size: 13,
-                        weight: .regular
-                    )
+                    NSAttributedString.Key.font: FXFontStyles.Regular.footnote.scaledFont()
```

Comment:
let's update this to use the body style. also if you can update the screenshot in the PR with the change, that would be great! thank you :)